extensions/pkg/terraformer/config.go

```diff
@@ -245,17 +246,29 @@ func (t *terraformer) CleanupConfiguration(ctx context.Context) error {
 	t.logger.Info("Cleaning up all terraformer configuration")
 
 	t.logger.V(1).Info("Deleting Terraform variables Secret", "name", t.variablesName)
-	if err := t.client.Delete(ctx, &corev1.Secret{ObjectMeta: metav1.ObjectMeta{Namespace: t.namespace, Name: t.variablesName}}); err != nil && !apierrors.IsNotFound(err) {
+	secret := &corev1.Secret{ObjectMeta: metav1.ObjectMeta{Namespace: t.namespace, Name: t.variablesName}}
+	if err := t.client.Delete(ctx, secret); err != nil && !apierrors.IsNotFound(err) {
```

Comment:
FYI: You can use a helper method for this:

```suggestion
	if err := t.client.Delete(ctx, secret); client.IgnoreNotFound(err) {
```